### PR TITLE
Input

### PR DESCRIPTION
**Pull request by mehul-anand**
# Input
### Label Inputs
- Standard
- Disabled 
- Error
### Radio Inputs
- Standard
- Disabled 
- Error
### Checkbox Inputs
- Standard
- Disabled 
- Error

### CODE REVIEW

File: card.css [hunk 1]
```diff
@@ -1,179 +1,155 @@
...
+    width: 100%;
+    height: 90%;
     object-fit: cover;
-
-} 
-
-.cardCollectionVerticalVariant{
+  }
+  
+  .cardCollectionVerticalVariant {
```
Comment (nayyyhaa) [inline card.css:30]: I'm not sure if camel-casing is suggested in CSS class names. I suggest you use hyphens or underscores. 
You can read more about it here: https://stackoverflow.com/questions/1547986/can-i-use-camelcase-in-css-class-names

Comment (mehul-anand) [reply card.css:30]: I'll use hyphens 
Thank you

File: card.css [hunk 2]
```diff
@@ -1,179 +1,149 @@
...
+  
+  .dismiss-icon {
     color: var(--primary-color);
-    cursor: pointer;  
-}
-
-.card-image{
-    width: 100%; 
-    height: 90%; 
+    cursor: pointer;
```
Comment (ShekharDhangar) [inline card.css:21]: use pointer as helper class

File: card.css [hunk 3]
```diff
@@ -1,179 +1,149 @@
...
+    border-radius: 0.25rem;
+  }
+  
+  .card-shadow {
+    box-shadow: 0.12rem 0.12rem 1.2rem 0.12rem var(--primary-color);
+  }
+  
+  .card-header {
     display: flex;
     justify-content: space-between;
```
Comment (ShekharDhangar) [inline card.css:16]: use flex as helper class and do custom styling in  the classes you are applying

File: card.css [hunk 4]
```diff
@@ -1,179 +1,149 @@
...
-}
-
-.container3{
+  }
+  
+  .card-text-container {
     /* background-color: var(--secondary-color);  */
     justify-content: center;
     align-items: center;
     display: flex;
```
Comment (ShekharDhangar) [inline card.css:60]: can use flex as helper class

**Review by ShekharDhangar — APPROVED** [review]
try to use as many helper class, it will stop repetitive code

**Pull request merged**